main.py

```diff
@@ -2,9 +2,38 @@
 
 from db.models import Race, Skill, Player, Guild
 
+import json
+
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        players_info = json.load(f)
+    for pl in players_info.keys():
```

Comment:
Consider using `players_info.items()` instead of `players_info.keys()` to directly iterate over key-value pairs, which can make the code cleaner and more efficient.